charts/trino/values.yaml

```diff
@@ -550,6 +550,21 @@ secretMounts: []
 # ```
 
 coordinator:
+  service: {}
+  # coordinator.service -- Service overrides just for the coordinator service. It is merged with the root-level service specification. Beware that specifying spec.ports will override the dynamically generated ports for JMX exporter as well as the additionalExposedPorts.
```

Comment:
IMO, we should not allow specifying both the top-level service and coordinator or worker services. It's too easy to make a mistake, like specifying the wrong type, and one will be ignored.

There's a similar PR https://github.com/trinodb/charts/pull/360 and we had this discussion there: https://github.com/trinodb/charts/pull/360#discussion_r2161014465